File: WhichPharmaPortal/WhichPharma.Models/Client/Supplier.cs
using System;

namespace WhichPharmaPortal.Models.Client
{
    public class Supplier
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public SupplierType Type { get; set; }
        public string Country { get; set; }
        public string CountryCode { get; set; }
        public bool IsEurope { get; set; }
        public int Classification { get; set; }
        public string Notes { get; set; }
        public Contact[] Contacts { get; set; }
        public SupplierState State { get; set; }
        public DateTime LastVerification { get; set; }
        public DateTime ExpirationDate { get; set; }
        public DateTime CreationDate { get; set; }
    }
}

File: WhichPharmaPortal/WhichPharmaPortal/ClientApp/src/models/Supplier.ts
import { SupplierType } from './SupplierType';
import { Contact } from './Contact';
import { SupplierState } from './SupplierState';

export interface Supplier {
    id: string;
    name: string;
    type: SupplierType;
    country: string;
    countryCode: string;
    isEurope: boolean;
    classification: number;
    notes: string;
    contacts: Contact[];
    state: SupplierState;
    lastVerification: string;
    expirationDate: string;
    creationDate: string;
}

export interface AGSupplier {
    data: {
        id: string;
        name: string;
        type: SupplierType;
        country: string;
        countryCode: string;
        isEurope: boolean;
        classification: number;
        notes: string;
        contacts: Contact[];
        state: SupplierState;
        lastVerification: string;
        expirationDate: string;
        creationDate: string;
    };
}
